src/components/structures/MessagePanel.js

```diff
@@ -388,6 +388,8 @@ module.exports = React.createClass({
                 isVisibleReadMarker = visible;
             }
 
+            // XXX: there should be no need for a ghost tile - we should just use a
```

Comment:
We could reuse the original marker and start the animation by calling `Velocity` on it before it moves. Instead of using a `ref` on the ghost, we'd use it on the original.